test/models/code_schools_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class CodeSchoolTest < ActiveSupport::TestCase
+  def setup
+    @code_school = build :code_school
+  end
+
+  test "is_partner should be present" do
+    @code_school.is_partner = true
+    assert @code_school.valid?
```

Comment:
As you updated the factory to set `is_partner` by default, you should be able to:

1. Create a new code school
2. Verify that `is_partner` is set to `false` (i.e. `assert @code_school.is_partner == false`)